src/test/java/org/fundacionjala/coding/yerel/WordOrderTest.java

```diff
@@ -0,0 +1,51 @@
+package org.fundacionjala.coding.yerel;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * this a test for class WordOrder.
+ */
+public class WordOrderTest {
+
+    /**
+     * test to inner content descending "wait for me".
+     */
+    @Test
+    public void innerContent() {
+        WordOrder wordOrder = new WordOrder();
```

Comment:
You are repeating this line of code in all the tests
Could you move it to a @before method